scripts/smoketest/smoketest.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path"
+	"path/filepath"
+	"strconv"
+	"syscall"
+)
+
+//TODO(jschiller)
+var remote = flag.String("remote", "", "Empty string to run locally. Otherwise: 'DC/ROLE/ENV'")
```

Comment:
Oh, this was actually meant for the twitter repo. I'm planning to do a corresponding script there with this flag.